src/plugins/zeromqrecv/CMakeLists.txt

```diff
@@ -2,7 +2,9 @@ include (LibAddBinding)
 
 if (DEPENDENCY_PHASE)
 	find_package (PkgConfig QUIET)
-	pkg_check_modules (ZMQ QUIET libzmq)
+	if (PKG_CONFIG_FOUND)
```

Comment:
Do we get correct error messages after this PR? Suppose someone has zmq installed but not pkg-config. Correct behavior should be that the error message says that `pkg-config is missing but needed to find zmq`. Is this the case?

But in general I like that the output gets more quiet.